OpenSim/Simulation/Model/Model.h

```diff
@@ -925,17 +925,17 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Model, ModelComponent);
     /**
      * Print some basic information about the model.
      *
-     * @param aOStream Output stream.
+     * @param aOStream Output stream (e.g., std::cout).
```

Comment:
Thanks for reviewing. Good point about the wrapping; I don't mind adding default `std::cout` arguments.